rest/routing.go

```diff
@@ -185,6 +185,9 @@ func CreateAdminRouter(sc *ServerContext) *mux.Router {
 	dbr.Handle("/_user/{name}",
 		makeHandler(sc, adminPrivs, []Permission{PermWritePrincipal}, nil, (*handler).deleteUser)).Methods("DELETE")
 
+	dbr.Handle("/_user/{name}/all_channels",
+		makeHandler(sc, adminPrivs, []Permission{PermReadPrincipal}, nil, (*handler).handleGetAllChannels)).Methods("GET", "HEAD")
```

Comment:
Needs changes to OpenAPI Specification to document this new API endpoint: `/docs/api/`

Reply:
Still need this.

Reply:
sry forgot it, fixed